src/Chessboard/helpers.js

```diff
@@ -186,6 +187,9 @@ function pieceCodeToFen(piece) {
   if (pieceCodeLetters[0] === 'w') {
     return pieceCodeLetters[1].toUpperCase();
   }
+  if (piece === '?') {
```

Comment:
what about the 0? I would like to see explicit matching for every case. That will make the code more readable (at least for me).
```suggestion
  if (piece === '?') {
  return '?';
  }
  if (piece === '0') {
  return '0';
  }
  if (pieceCodeLetters[0] === 'w') {
    return pieceCodeLetters[1].toUpperCase();
  }
  if (pieceCodeLetters[0] === 'b') {
    return pieceCodeLetters[1].toLowerCase();
  }
```

Reply:
I think change 0 or ? to "occupied" and "unknown" text make code more complicated, cause at the and I will add the oder code that should do the opposite for sending you fen. In future we can add some tooltip on piece which can describe the piece.